Cython/Utility/ObjectHandling.c

```diff
@@ -2214,10 +2214,39 @@ static PyObject* __Pyx_PyObject_CallMethod1(PyObject* obj, PyObject* method_name
 
 
 /////////////// tp_new.proto ///////////////
+//@requires: ModuleSetupCode.c::GetRuntimeVersion
 
 #define __Pyx_tp_new(type_obj, args) __Pyx_tp_new_kwargs(type_obj, args, NULL)
 static CYTHON_INLINE PyObject* __Pyx_tp_new_kwargs(PyObject* type_obj, PyObject* args, PyObject* kwargs) {
```

Comment:
This function is inlined and used to be short, which is the reason why i lives entirely in a `proto` section.

I suggest moving the fallback code to a separate non-inline function and putting that in the normal impl (no suffix) utility code section, to move it clearly out of the fast paths and after the user code in the C file.